core/src/main/java/nl/nn/adapterframework/receivers/Receiver.java

```diff
@@ -1391,7 +1394,9 @@ private synchronized ProcessResultCacheItem getCachedProcessResult(String messag
 	 */
 	@SuppressWarnings("unchecked")
 	private boolean hasProblematicHistory(String messageId, boolean manualRetry, Object rawMessageOrWrapper, Message message, Map<String,Object>threadContext, String correlationId) throws ListenerException {
-		if (!manualRetry) {
+		if (manualRetry) {
+			threadContext.put("retry", "true");
```

Comment:
Ik weet niet precies waar dit uitgelezen wordt, maar aangezien we het nu meerdere malen aan het gebruiken zijn, kan hier een constante van gemaakt worden?